pallets/simple_democracy/src/lib.rs

```diff
@@ -0,0 +1,334 @@
+        fn remove_other_vote(origin, target: T::AccountId, ref_index: ReferendumIndex) -> DispatchResult {
+            let who = ensure_signed(origin)?;
+            let scope = if target == who { UnvoteScope::Any } else { UnvoteScope::OnlyExpired };
+            Self::try_remove_vote(&target, ref_index, scope)
+        }
+
+        /// Proxy function to forked democracy pallet's `enact_proposal`
+        #[weight = T::MaximumBlockWeight::get()]
+        fn enact_proposal(origin, proposal_hash: T::Hash, index: ReferendumIndex) -> DispatchResult {
+            <pallet_democracy::Module<T>>::enact_proposal(origin, proposal_hash, index)
+        }
+
+        /// Proxy function to forked democracy pallet's `note_preimage`
+        // TODO: Fix weight
+        #[weight = 0]
+        pub fn note_preimage(origin, encoded_proposal: Vec<u8>) {
+            <pallet_democracy::Module<T>>::note_preimage(origin, encoded_proposal)?;
+        }
+
+        /// Proxy function to forked democracy pallet's `note_preimage_operational`
+        // TODO: Fix weight
+        #[weight = (0, DispatchClass::Operational)]
+        fn note_preimage_operational(origin, encoded_proposal: Vec<u8>) {
+            <pallet_democracy::Module<T>>::note_preimage_operational(origin, encoded_proposal)?;
+        }
+
+        /// Proxy function to forked democracy pallet's `reap_preimage`
+        // TODO: Fix weight
+        #[weight = 0]
+        pub fn reap_preimage(origin, proposal_hash: T::Hash, #[compact] proposal_len_upper_bound: u32) {
+            <pallet_democracy::Module<T>>::reap_preimage(origin, proposal_hash, proposal_len_upper_bound)?;
+        }
+
+        /// Proxy function to forked democracy pallet's `cancel_proposal`
+        // TODO: Fix weight
+        #[weight = 0]
+        fn cancel_proposal(origin, #[compact] prop_index: PropIndex) {
+            <pallet_democracy::Module<T>>::cancel_proposal(origin, prop_index)?;
+        }
+
+        /// Proxy function to forked democracy pallet's `cancel_queued`
+        // TODO: Fix weight
+        #[weight = (0, DispatchClass::Operational)]
+        fn cancel_queued(origin, which: ReferendumIndex) {
+            <pallet_democracy::Module<T>>::cancel_queued(origin, which)?;
+        }
+
+        /// Proxy function to forked democracy pallet's `cancel_referendum`
+        // TODO: Fix weight
+        #[weight = 0]
+        fn cancel_referendum(origin, #[compact] ref_index: ReferendumIndex) {
+            <pallet_democracy::Module<T>>::cancel_referendum(origin, ref_index)?;
+        }
+
+        /// Proxy function to forked democracy pallet's `clear_public_proposals`
+        // TODO: Fix weight
+        #[weight = 0]
+        fn clear_public_proposals(origin) {
+            <pallet_democracy::Module<T>>::clear_public_proposals(origin)?;
+        }
+
+        /// Proxy function to forked democracy pallet's `on_initialize`
+        // TODO: Set weight
+        fn on_initialize(n: T::BlockNumber) -> Weight {
```

Comment:
This will result in `<pallet_democracy::Module<T>>::on_initialize(n)` being called twice every block.

Reply:
Thanks. Fixed.